### PR TITLE
complete issue 146

### PR DESCRIPTION
**Pull request by herecomesjaycee**
@womenhackfornonprofits/whfnp
## Linked Jira/Github Issue Ticket

https://github.com/womenhackfornonprofits/whfnp/issues/146
## What are the changes included in this PR?

Edit volunteer.html to include bootstrap grid
## If the changes are visual, provide a screenshot.

Visual screenshots are provided on the issue146 ticket

### CODE REVIEW

**Review by VAO11 — COMMENTED** [review]
Hey Jaycee, Great work. he ony thing I noticed was the spacing of the how it works section. It has changed from the  original view.

image

I think this can resolved by replacing the div class = "col-wrap" (css)  with div class="container" (bootstrap).

**Review by VAO11 — COMMENTED** [review]
Great! It back to normal now.

File: src/volunteers.html [hunk 1]
```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
...
+  <div class="row">
+    <div class="col-sm-6 col-md-5 col-md-offset-2 col-lg-6 col-lg-offset-0">
       <a href="http://marizadima.wordpress.com" class="volunteer-link">
         <img class="volunteer-picture" src="img/volunteers/Mariza_Dima.png" alt="Mariza Dima Picture">
         <h2>Mariza Dima</h2>
       </a>
       <p class="volunteer-quote">"It's a rewarding experience to work as a tech developer for good causes and even more so in collaboration with women. In WHFNP we bring together strong expertise and project management with a feminine perspective in providing solutions to complex problems, and designing applications that can make a change."</p>
     </div>
-    <div class="col-2">
+    <div class="col-sm-6 col-md-5 col-md-offset-2 col-lg-6 col-lg-offset-0">
```
Comment (lili2311) [inline src/volunteers.html]: Why do we need `col-lg-offset-0`? That does an offset of 0 columns, can it be removed unless it does something special?

File: src/volunteers.html [hunk 2]
```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
 <section>
   <div class="container">
     <h2 class="section-heading">How it works</h2>
-    <div class="col-wrap">
-      <div class="col-3 centered-text">
+    <div class="container">
+      <div class="col-md-4 centered-text" align="center">
```
Comment (lili2311) [inline src/volunteers.html]: align="center" what does this do? This is not valid inline css (also please no inline css, all css to be in the css files), please can we the bootsrap centering class.

File: src/volunteers.html [hunk 3]
```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
...
         <img class="non-profit-icons" src="img/icons/email-us.svg" alt="Email Us Image">
         <h2 class="section-heading">1. Contact Us</h2>
         <p>Fill in the <a class="text-link" href="volunteer-contact.html">volunteer form</a> and hit send.</p>
       </div>
-      <div class="col-3">
+      <div class="col-md-4" align="center">
```
Comment (lili2311) [inline src/volunteers.html]: same here

File: src/volunteers.html [hunk 4]
```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
...
         <img class="non-profit-icons" src="img/icons/email-confirmation.svg" alt="Email Confirmation Image">
         <h2 class="section-heading">2. Confirmation</h2>
         <p>We will email you all the info you need to get started.</p>
       </div>
-      <div class="col-3">
+      <div class="col-md-4" align="center">
```
Comment (lili2311) [inline src/volunteers.html]: same here

File: src/volunteers.html [hunk 5]
```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
...
+        </div>
       </div>
     </div>
-  </div>
+
 </section>
 <section>
   <h2 class="section-heading">Meet our volunteers</h2>
   <p>Meet some of our volunteers to find out what you could be both contributing and learning by joining WHFNP.</p>
   <div class="volunteers-wrap">
```
Comment (lili2311) [inline src/volunteers.html]: `volunteer-wrap` doesn't exist in the css, please remove this and the div

File: src/volunteers.html [hunk 6]
```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
...
-    <div class="col-2">
+  <div class="row">
+    <div class="col-sm-6 col-md-5 col-md-offset-2 col-lg-6 col-lg-offset-0">
```
Comment (lili2311) [inline src/volunteers.html]: `col-lg-offset-0` here as well

File: src/volunteers.html [hunk 7]
```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
...
-    <div class="col-2">
+    <div class="col-sm-6 col-md-5 col-md-offset-2 col-lg-6 col-lg-offset-0">
       <a href="http://echesters.github.io" class="volunteer-link">
         <img class="volunteer-picture" src="img/volunteers/Elizabeth_Chesters.png" alt="Elizabeth Chesters Picture">
         <h2>Elizabeth Chesters</h2>
       </a>
       <p class="volunteer-quote">"I joined WHFNP to help me grow in a new career path into user experience, and to expand my portfolio. I also love having side projects with a full team, which gives me motivation and no excuses to procrastinate! The projects have amazing meanings behind them, each with their unique challenges."</p>
     </div>
-    <div class="col-2">
+    </div>
```
Comment (lili2311) [inline src/volunteers.html]: Please can the HTML be formatted with correct spacing? usually 4 spaces, this is a good place to use: http://www.cleancss.com/html-beautify/ or in the text editor can use the spacing it provides or a [linter](https://github.com/SublimeLinter/SublimeLinter-html-tidy).

File: src/volunteers.html [hunk 8]
```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
...
+    <div class="row">
+    <div class="col-sm-6 col-md-5 col-md-offset-2 col-lg-6 col-lg-offset-0">
```
Comment (lili2311) [inline src/volunteers.html]: `col-lg-offset-0` here

File: src/volunteers.html [hunk 9]
```diff
@@ -11,57 +11,62 @@ <h1 class="title-heading title-cta">Volunteers</h1>
...
       <a href="http://annaleach.net/" class="volunteer-link">
-        <img class="volunteer-picture" src="img/volunteers/Anna_Leach.png" alt="Anna Leach Picture">
+        <img class="volunteer-picture" src="img/volunteers/Anna_Leach.png" alt="Anna Leach Picture" >
```
Comment (lili2311) [inline src/volunteers.html]: extra space at the end of the html tag, please remove

**Review by lili2311 — CHANGES_REQUESTED** [review]

File: src/volunteers.html [hunk 10]
```diff
@@ -1,78 +1,67 @@
...
-</section>
-<section>
-  <h2 class="section-heading">Start your journey here</h2>
-  <p>Excited to get started? We’re excited to hear from you! Please fill in the form below to join the WHFNP community.</p>
-  <a href="volunteer-contact.html" class="button">Volunteer with us</a>
-</section>
+    --- layout: default title: Volunteers | Women Hack for Non-Profits ---
+    <section class="smaller-banner volunteers-image">
+        <div class="container">
+            <h1 class="title-heading title-cta">Volunteers</h1><a class="button" href="volunteer-contact.html">Volunteer with us</a>
```
Comment (lili2311) [inline src/volunteers.html]: `title-cta` is not a class anywhere else and has no entry in the css files, please remove.

**Review by lili2311 — CHANGES_REQUESTED** [review]

**Review by lili2311 — APPROVED** [review]

**Comment by lili2311** [comment]
👍 perfect ty! merging

**Pull request merged**